### PR TITLE
[cluster-test] increase test duration for fullnode recovery

### PR DESCRIPTION
**Pull request by snakech**
## Motivation

(Write your motivation for proposed changes here.)

### Have you read the [Contributing Guidelines on pull requests](https://github.com/libra/libra/blob/master/CONTRIBUTING.md#pull-requests)?

(Write your answer here.)

## Test Plan

(Share your test plan here. If you changed code, please provide us with clear instructions for verifying that your changes work.)

## Related PRs

(If this PR adds or changes functionality, please take some time to update the docs at https://github.com/libra/libra/developers.libra.org, and link to your PR here.)

### CODE REVIEW

**Comment by andll** [comment]
Do you mind giving some context/motivation on this change?

**Comment by snakech** [comment]
> Do you mind giving some context/motivation on this change?

https://toroeng.slack.com/archives/CNJCQD2J0/p1601917065000100
10% down test occasionally failed last few days, checked failed job which shows post test healthcheck failed, checked grafana which fullnode did not recovery in time. Try to increase the duration first to see whether test can back to stable, for next step we can extent THRESHOLD back to 20k in fullnode health check

**Review by andll — APPROVED** [review]

File: testsuite/cluster-test/src/experiments/performance_benchmark.rs [hunk 1]
```diff
@@ -265,7 +265,7 @@ impl Experiment for PerformanceBenchmark {
     }
 
     fn deadline(&self) -> Duration {
-        Duration::from_secs(600) + self.duration
+        Duration::from_secs(900) + self.duration
```
Comment (sausagee) [inline testsuite/cluster-test/src/experiments/performance_benchmark.rs:268]: Should we make this limit a bar for future regression?

**Pull request closed without merging**